engine/src/flutter/lib/web_ui/skwasm/images_skia.cpp

```diff
@@ -0,0 +1,154 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "images.h"
+
+#include "flutter/display_list/geometry/dl_geometry_conversions.h"
+#include "flutter/display_list/skia/dl_sk_conversions.h"
+#include "flutter/display_list/skia/dl_sk_dispatcher.h"
+#include "third_party/skia/include/core/SkColorSpace.h"
+#include "third_party/skia/include/core/SkData.h"
+#include "third_party/skia/include/core/SkImage.h"
+#include "third_party/skia/include/core/SkImageInfo.h"
+#include "third_party/skia/include/core/SkPicture.h"
+#include "third_party/skia/include/core/SkPictureRecorder.h"
+#include "third_party/skia/include/gpu/GpuTypes.h"
+#include "third_party/skia/include/gpu/ganesh/GrBackendSurface.h"
+#include "third_party/skia/include/gpu/ganesh/GrDirectContext.h"
+#include "third_party/skia/include/gpu/ganesh/GrExternalTextureGenerator.h"
+#include "third_party/skia/include/gpu/ganesh/SkImageGanesh.h"
+#include "third_party/skia/include/gpu/ganesh/SkSurfaceGanesh.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLBackendSurface.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLInterface.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLTypes.h"
+
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+#include <emscripten/html5_webgl.h>
+
+using namespace flutter;
+using namespace SkImages;
+using namespace Skwasm;
+
+namespace {
+
+SkColorType colorTypeForPixelFormat(PixelFormat format) {
```

Comment:
Improper naming of function: https://google.github.io/styleguide/cppguide.html#Function_Names

(PascalCase)